# Disable SSLv2, SSLv3, and TLSv1.
# Upstream has disabled SSLv2 and SSLv3. Disabling of TLSv1 will not go upstream

--- irssi-0.8.15/src/core/network-openssl.c.orig	Wed Dec 16 10:30:52 2015
+++ irssi-0.8.15/src/core/network-openssl.c	Wed Dec 16 10:30:27 2015
@@ -406,6 +406,7 @@
 		g_error("Could not allocate memory for SSL context");
 		return NULL;
 	}
+	SSL_CTX_set_options(ctx, SSL_OP_NO_SSLv2 | SSL_OP_NO_SSLv3 | SSL_OP_NO_TLSv1);
 
 	if (mycert && *mycert) {
 		char *scert = NULL, *spkey = NULL;
